tools/ios/MASTG-TOOL-0102.md

```diff
@@ -0,0 +1,12 @@
+---
+title: ios-app-signer
+platform: ios
+source: https://github.com/DanTheMan827/ios-app-signer
+---
+
+[ios-app-signer](https://github.com/DanTheMan827/ios-app-signer) is a GUI-based tool for macOS that allows you to re-sign iOS apps and bundle them into IPA files ready to be installed on an iOS device. It can be installed using the `.app` file from the [GitHub releases](https://github.com/DanTheMan827/ios-app-signer/releases) or via brew on macOS:
```

Comment:
```suggestion
[ios-app-signer](https://github.com/DanTheMan827/ios-app-signer) is a GUI-based tool for macOS that allows you to re-sign iOS apps and bundle them into IPA files ready to be installed on an iOS device. It can be installed using the `.app` file from the [GitHub releases](https://github.com/DanTheMan827/ios-app-signer/releases) or via brew on macOS:

```